tests/functional/firefox-sync.js

```diff
@@ -0,0 +1,122 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global casper */
+'use strict';
+
+var config = require('../lib/config');
+var helpers = require('../lib/helpers');
+var path = '/firefox/sync/';
+var url = config.base() + path;
+
+casper.test.begin('Firefox Sync, Elements: ' + url, 7, function suite(test) {
+
+    casper.start(url, function() {
+        test.assertHttpStatus(200);
+
+        test.assertVisible('#newsletter-form', 'Newsletter form visible');
+        test.assertVisible('.cta', 'The CTA is visible');
+        test.assertVisible('#cta-android-footer', 'Download Fx for Android button visible');
+        test.assertElementCount('.secondary .features div', 3, 'List of three feature exists');
+    });
+
+    casper.waitUntilVisible('.sync-anim.on', function() {
```

Comment:
Would `waitForSelector` be a better fit here? git pull https://github.com/schalkneethling/bedrock.git casper-tests-firefox-sync